server/src/main/java/org/elasticsearch/health/HealthPeriodicLogger.java

```diff
@@ -81,9 +114,13 @@ public class HealthPeriodicLogger implements ClusterStateListener, Closeable, Sc
     private final SetOnce<SchedulerEngine> scheduler = new SetOnce<>();
     private volatile TimeValue pollInterval;
     private volatile boolean enabled;
+    private volatile OutputMode outputMode;
 
     private static final Logger logger = LogManager.getLogger(HealthPeriodicLogger.class);
 
+    private final MeterRegistry meterRegistry;
+    private final Map<String, LongGaugeMetric> metrics = new HashMap<>();
```

Comment:
This is a map for all *red* metrics, so maybe we should call it `redMetrics` so it's clear that it's not for green or yellow metrics?